ca/ca.go

```diff
@@ -492,8 +493,24 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 		return emptyCert, err
 	}
 
+	var ocspResp []byte
+	if features.Enabled(features.GenerateOCSPEarly) {
+		ocspResp, err = ca.GenerateOCSP(ctx, core.OCSPSigningRequest{
+			CertDER: certDER,
+			Status:  "good",
+		})
+		if err != nil {
+			err = berrors.InternalServerError(err.Error())
+			ca.log.AuditInfo(fmt.Sprintf("OCSP Signing failure: serial=[%s] pem=[%s] err=[%s]",
+				serialHex, certPEM, err))
+			// If OCSP generation fails don't return an error and orphan the
```

Comment:
If we returned the `err` here it would have the same meaning. The reason I stuck a comment here was because we are extremely paranoid about doing proper error checking in this function and we are basically explicitly ignoring an error here so I think a warning of some kind is probably necessary. Do you have thoughts on better wording?